corehq/apps/hqadmin/templates/hqadmin/email/mass_email_base.txt

```diff
@@ -0,0 +1 @@
+{{ email_body|striptags }}
```

Comment:
what happens to URLs when you do this? If you have something like `<a href='example.com'>click here!</a>` won't this just say "click here!"?

Reply:
I guess this isn't important since you are pasting the text content in the form.

Reply:
Yes, that's what will happen. User needs to know not to put tags in their plain text content.